cobalt/h5vcc/h5vcc_settings.cc

```diff
@@ -67,35 +57,37 @@ bool H5vccSettings::Set(const std::string& name, int32 value) const {
   const char kUpdaterMinFreeSpaceBytes[] = "Updater.MinFreeSpaceBytes";
 #endif
 
-  if (name == kDisableMediaCodec &&
-      value < static_cast<int32>(kDisableCodecCombinations.size())) {
-    can_play_type_handler_->SetDisabledMediaCodecs(
-        kDisableCodecCombinations[value]);
+  if (name == kMediaCodecBlockList && value.IsType<std::string>() &&
+      value.AsType<std::string>().size() < 256) {
+    can_play_type_handler_->SetDisabledMediaCodecs(value.AsType<std::string>());
     return true;
   }
 
```

Comment:
nit: If we add the following code here, we can save the all the checking below.  Feel free to keep everything as is though.
```
if (!value.IsType<int32>()) {
  return false;
}
```

Reply:
Agree. But I think the current design is easier to maintain where we need to explicitly check what type of value we input to the console.